compiled/tutorial/manifests/echo-server-bundle.yml

```diff
@@ -94,7 +94,7 @@ spec:
       volumes:
         - configMap:
             defaultMode: 550
-            name: echo-server-b467a43a
```

Comment:
this changes because by mistake we were hashing the whole object (including python calls) rather than just the output of object.root